app/policies/churn_policy.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ChurnPolicy < ApplicationPolicy
+  def show?
+    has_subscription_products = seller.products.alive.is_recurring_billing.exists?
```

Comment:
A user with membership products that are no longer alive should still be able to access the page to see historical data.
Also, it would be a better UX for a user without any membership products to be able to access this page and see a placeholder with information about the Churn dashboard.